openslides/agenda/projector.py

```diff
@@ -64,3 +64,32 @@ def need_full_update_for_this(self, collection_element):
         # Full update if item changes because then we may have new speakers
         # and therefor need new users.
         return collection_element.collection_string == Item.get_collection_string()
+
+
+class CurrentListOfSpeakersSlide(ProjectorElement):
+    """
+    Slide for the current list of speakers.
+
+    Nothing special to check.
+    """
+    name = 'agenda/current-list-of-speakers'
+
+    def get_requirements(self, config_entry):
+        pk = config['projector_currentListOfSpeakers_reference']
```

Comment:
@emanuelschuetze mentioned that this should be a different feature. I think like together with the changes in `isProjected()` for supporting one slide on multiple projectors.